packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/java/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,98 @@
+### 1: Build a sign-in page on the client
+
+Build a sign-in page that captures the user's name and password, as shown in the following example.
+
+![Basic sign-in dialog](/img/authenticators/java-authenticators-signinform.png)
+
+### 2: Authenticate the user credentials
+
+After a user initiates the sign-in flow by entering their username and password and then clicking **Sign In**, add the following code:
+
+```java
+// create an IDX Authentication Wrapper (can be an application scoped singleton)
+IDXAuthenticationWrapper idxAuthenticationWrapper = new IDXAuthenticationWrapper();
+
+// begin transaction
+// The proceedContext needs to be persisted between interactions (for example, stored in a HttpSession for web apps)
+ProceedContext proceedContext = idxAuthenticationWrapper.begin().getProceedContext();
+
+// set the user's credentials
+AuthenticationOptions authenticationOptions = new AuthenticationOptions(username, password);
+
+// start authentication flow
+AuthenticationResponse authenticationResponse = idxAuthenticationWrapper.authenticate(authenticationOptions, proceedContext);
+```
+
+### 3: Handle the response from the sign-in flow
+
+The `AuthenticationResponse` contains the current state of the authentication process. It could be a `SUCCESS` or may indicate that more information is needed to complete the flow, for example, require an additional factor.
+
+Query the `authenticationStatus` property of the `AuthenticationResponse` object returned by `authenticate` to discover the current status of the authentication process.
+
+```java
+// Update the context each time a request is made to Okta
+// make sure it is persisted for web applications
+proceedContext = authenticationResponse.getProceedContext();
+
+// get the authenticationStatus
+AuthenticationStatus authenticationStatus = authenticationResponse.getAuthenticationStatus();
+
+switch (authenticationStatus) {
+    case SUCCESS:
+        … your code …
+    case PASSWORD_EXPIRED:
+        … your code …
+}
+```
+
+### 4: Select a possible authenticator factor
+
+If you configure your Okta org correctly, you need to respond to two specific authenticator statuses to handle this scenario in addition to `Success` and `PasswordExpired`:
```

Comment:
This looks like a leftover from the .net version here. Are the statuses called Success and PasswordExpired in Java?